devbox_test.go

```diff
@@ -48,15 +48,16 @@ func testExample(t *testing.T, testPath string) {
 		box.srcDir, err = filepath.Rel(currentDir, box.srcDir)
 		assert.NoErrorf(err, "expect to construct relative path from %s relative to base %s", box.srcDir, currentDir)
 
-		plan, err := box.ShellPlan()
+		plan, err := box.BuildPlan()
+		buildPlanErrorFile := filepath.Join(baseDir, "has_build_error")
+		hasBuildErrorFile := fileExists(buildPlanErrorFile)
+		if hasBuildErrorFile {
+			return
```

Comment:
Instead of just returning, shouldn't we be asserting that there _is_ an error?

Reply:
oh ummm ok I'll rename the file. It should be `build_error_expected` or `build_should_error_and_it_should_not_succeed`